sympy/solvers/solveset.py

```diff
@@ -882,6 +882,101 @@ def solveset_complex(f, symbol):
     return solveset(f, symbol, S.Complexes)
 
 
+def solveset_integers(f, symbols = None, param = Symbol("t", integer = True)):
+    r"""Solves a given equation in domain S.Integers. `diophantine`, defined in
+    solvers/diophantine.py; is used to get the integer solution..
+
+    Parameters
+    ==========
+
+    f : Expr or a relational.
+        The target equation
+    symbols : list of Symbol
+        The variable for which the equation is solved. The order of
+        symbols in input `symbols` will determine the order of values
+        in the solution tuple.
+    param : symbol that should be used in integer solution
+
+    Returns
+    =======
+
+    Set
+        A FiniteSet, if finite number of solution is present otherwise
```

Comment:
Can you paraphrase the description for clarity. Using shorter sentences might help.